tools/history-migration/src/kilda/history_migration/cli.py

```diff
@@ -0,0 +1,152 @@
+#  Copyright 2021 Telstra Open Source
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import collections
+import datetime
+
+import click
+import mysql.connector
+import pyorient
+
+from . import orphaned
+from . import pull_mysql
+from . import pull_orient
+from . import push
+
+
+@click.group()
+def cli():
```

Comment:
what is this used for?

Reply:
Is path of "click" api, required to create command/options group.